.github/workflows/test_frontend.yml

```
@@ -24,6 +24,12 @@ jobs:
       - name: setup env variables
         working-directory: ./frontend
         run: cp .env.example .env
+      - name: lint
+        working-directory: ./frontend
+        run: yarn lint
+      - name: format
+        working-directory: ./frontend
+        run: yarn checkformat
```

Der Job check gerade nur das Format, aber formatiert nicht, deshalb würde ich den Namen ergänzen
```suggestion
      - name: check format
        working-directory: ./frontend
        run: yarn checkformat
```